### PR TITLE
Point a new woopra repo. Remove tarball

### LINKED ISSUE #656: wpt
Hi,
Regarding `wpt` being pulled from Github, could you maybe inline this package instead of relying on Github?
Thank you

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
This PR should fix #656 and #726

# Solution
 The solution involves removing tarball and make woopra a dev dependency

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [x] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +20,9 @@ install:
   - npm --production=false install
 script:
   - npm run build
+  - rvm install "ruby-2.4.1"
+  - rvm use "ruby-2.4.1"
+  - gem install bundler -v 1.17.3
```
Comment (stefaniacardenas) [inline .travis.yml]: We also have gem install `bundler -v '< 2'` in `scripts/setup_test_environment.sh`, do we need to install it twice?

Comment (anuragajwani) [reply .travis.yml]: @stefaniacardenas does that script run on Travis builds?

Comment (stefaniacardenas) [reply .travis.yml]: Yes, check line 18 of this file

File: test.sh [hunk 2]
```diff
@@ -5,13 +5,7 @@ set -e
...
 if [[ ${CI} != "true" || (${NODE_ENV} = "test" && ${TRAVIS_PULL_REQUEST} != "false") ]]; then
   # set path for test directory
-
-  # setup to run browser GUI
-  if [[ ${CI} = "true" ]]; then
-    export DISPLAY=:99.0
-    sh -e /etc/init.d/xvfb start &
-    sleep 3 # give xvfb some time to start
-  fi
+  bundler --version
```
Comment (stefaniacardenas) [inline test.sh]: Is this doing anything? Is the comment above still valid?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

File: .travis.yml (hunk 1 above)
Comment (rfreitas) [reply .travis.yml]: travis failed telling me that bundler was not installed. this is why I addded it

**Comment by rfreitas** [comment]
Travis automatic deployment: https://731-pr-onfido-sdk-ui-onfido.surge.sh

File: test.sh (hunk 2 above)
Comment (rfreitas) [reply test.sh]: the comment above is leftover from before, not from now. but I can remove it.
the bundler --version is a quick log to let us know which version travis is running. I think is a good idea to leave it

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

File: .travis.yml (hunk 1 above)
Comment (stefaniacardenas) [reply .travis.yml]: I see... Maybe something changed on travis, because before it was able to install it from the script file.

Comment (stefaniacardenas) [reply .travis.yml]: Do we need to keep it in both places then?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://731-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-731-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**